README.md

```diff
@@ -1,49 +1,73 @@
+| GOOGLE_TOTP_SECRET   | If the Google account has 2FA security, then the TOTP secret that is configured for 2FA goes here.          | NA                                                  |
+| HTTP_PORT            | Port on which the meetbot server starts running. For balena devices, the server needs to run on port 80.    | 8080                                                |
+| MAX_BOTS             | Maximum number of meetbots to run parallely on the server                                                   | 5                                                   |
+| GREETING_MESSAGE     | Greeting message which is posted when meetbot joins the Google Meet                                         | "Hello balenistas, it's your favorite bot, meetbot" |
+
+## Getting Started
+
+After getting the bot server [up and running](#deployment), it will start listening for requests to provision new meetbots. 
 
-The bot will now be running but functionality is limited until you add additional configurations for [authentication](#authentication).
+To get a meetbot in your meeting, navigate to the Meetbot dashboard using the [public device URL](https://www.balena.io/docs/learn/manage/actions/#enable-public-device-url) or the port you configured for the server to run on. 
 
-To get a bot to join a Google Meet, send a POST request to the join endpoint with the meet url as mentioned below:
+Click the "Join Meeting" button and enter the Google Meet URL in the text box. Click "Join Meeting" and wait for a few seconds for the bot to join. When the bot joins the meeting or prompts to join the meeting, it will post a message to the chat to signal that's it's ready to go.
+
+### Running the bot locally
+
+After cloning the repository, install the dependencies:
 
 ```
-curl --header "Content-Type: application/json" \
-  --request POST \
-  --data '{"url":"https://meet.google.com/wtq-bhai-amg"}' \
-  http://localhost:8080/api/join
+npm ci
+npm start
 ```
-## UI Development
-To develop the UI, run `npm run dev-ui`. This will spin up the development server. You should then be able to access the dashboard at `http://localhost:3000/`. Note that the data you will see on the tables are actually being mocked by MirageJS, and not actual request is going to the API. This makes it easier to develop both components in an entirely decoupled way.
 
-A meetbot should request to join the Google Meet.
+The bot will now be running but functionality is limited until the bot is [authenticated](#authentication). To get a bot to join a Google Meet, head to the Meetbot dashboard available on `http://localhost:8080` and click the `Join` button. Copy the Google Meet URL and paste it into the text field.
+
 
 ## Authentication
 
-Copy `.env.example` to `.env` and populate it with the correct information. This is used by the bot to login to accounts.google.com.
+Authentication is needed to:
```

Comment:
After you removed instructions, how does the user know how to set up authentication? 
Even with the previous wording I would still be wondering how to get the TOTP token, as that's definitely not straight forward

Reply:
The authentication section definitely needs more work, @somombo is working to simplify it.

Reply:
We need to provide a lot of context, and I am considering a new page for authentication

Reply:
Added more information but still needs work.